README.md

```diff
@@ -102,6 +102,7 @@ apiVersion: 'kubernetes-client.io/v1'
 kind: ExternalSecret
 metadata:
   name: hello-service
+  namespace: default
```

Comment:
No need for this.

Reply:
Hey, I added this so it is more clear that you can also specify another namespace: 
>small nitpick since it's not mentioned anywhere but supported, adding namespace to the example as an implicit option

Reply:
I'd keep this out and still give the user the possibility to install this in a whatever namespace using the `--namespace` arg.